web/webpack.config.ts

```diff
@@ -375,6 +375,96 @@ module.exports = {
             }
           }
         },
+        {
+          type: "console.tab/horizontalNav",
+          properties: {
+            model: {
+              version: "v1",
+              group: "gateway.networking.k8s.io",
+              kind: "Gateway"
+            },
+            component: {
+              "$codeRef": "netflowTab.default"
+            },
+            "page": {
+              name: "%plugin__netobserv-plugin~Network Traffic%",
+              "href": "netflow"
+            }
+          },
+          "flags": { "required": ["NETOBSERV_LOKI_ENABLED"] }
```

Comment:
I don't think we need the loki enabled flag here: the Gateways should be set as Owners, which is known to prometheus, so that should work with loki disabled I think?